### PR TITLE
[14.0][MIG] account_invoice_fiscal_position_update: Migration to 14.0

### PR DESCRIPTION
**Pull request by xavierjimenez**
Migration to 14.0 of the module account_invoice_fiscal_position_update

### CODE REVIEW

**Comment by xavierjimenez** [comment]
Hi @MiquelRForgeFlow could you do a review if you can please?

Thanks!

File: account_invoice_fiscal_position_update/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2011-2014 Julius Network Solutions SARL <contact@julius.fr>
+# Copyright 2014-2016 Akretion (Alexis de Lattre
+#   <alexis.delattre@akretion.com>)
+# Copyright 2018 Roel Adriaans <roel@road-support.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+{
+    "name": "Invoice Fiscal Position Update",
+    "version": "14.0.1.0.1",
```
Comment (MiquelRForgeFlow) [inline account_invoice_fiscal_position_update/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

File: account_invoice_fiscal_position_update/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "category": "Accounting & Finance",
+    "license": "AGPL-3",
+    "summary": "Changing the fiscal position of an invoice will auto-update "
```
Comment (MiquelRForgeFlow) [inline account_invoice_fiscal_position_update/__manifest__.py]:
```suggestion
    "summary": "Changing the fiscal position of an invoice will auto-update."
```

**Review by MiquelRForgeFlow — APPROVED** [review]
I am not into v14 yet, but AFAIK, code LGTM

**Review by anasuarez1 — APPROVED** [review]

**Review by anasuarez1 — COMMENTED** [review]
LGTM!

File: account_invoice_fiscal_position_update/__manifest__.py (hunk 1 above)
Comment (xavierjimenez) [reply account_invoice_fiscal_position_update/__manifest__.py]: Changed!

File: account_invoice_fiscal_position_update/__manifest__.py (hunk 2 above)
Comment (xavierjimenez) [reply account_invoice_fiscal_position_update/__manifest__.py]: Changed!

**Review by chusamo — APPROVED** [review]
Tested in Local LGTM.

**Comment by MiquelRForgeFlow** [comment]
Please @xavierjimenez , cherry-pick https://github.com/OCA/account-invoicing/commit/9193a522666e71373ab0acd094b326349248934d.

**Comment by romi477** [comment]
Hi guys, I wanted to know when this one will be merged into 14.0 branch?

**Comment by alexis-via** [comment]
I'm sorry, but this PR doesn't work. Porting the module account_invoice_fiscal_position_update is MUCH MORE COMPLEX than that! In the current implementation, the total tax of the invoice is not updated.

**Comment by alexis-via** [comment]
I made a PR to migrate the module to v14 which seems to work (at last !) : https://github.com/OCA/account-invoicing/pull/963
As you can see in my PR, migrating this module is complex. The current version in v13 doesn't really work.

**Comment by HaraldPanten** [comment]
Superseded by https://github.com/OCA/account-invoicing/pull/963

**Pull request closed without merging**